doc/examples/example1.md

```diff
@@ -0,0 +1,139 @@
+Example 1 {#example1}
```

Comment:
Change the example title so the contents of the example are clear from the doxygen index. Maybe something like "Asymmetric Key Example"